### PR TITLE
Set map height for mobile resolutions

### PR DESCRIPTION
**Pull request by dutraneto**
@marceloglacial, please review issue #24. Appreciated.

### CODE REVIEW

**Review by marceloglacial — CHANGES_REQUESTED** [review]
Header overlapping the address when you click on the nav to jump to "localicação"

Screen Shot 2021-03-09 at 9 14 41 PM

Screen Shot 2021-03-09 at 9 13 40 PM

**Comment by dutraneto** [comment]
I will take a look right now!

**Comment by dutraneto** [comment]
Screen Shot 2021-03-09 at 9 24 34 PM

Hey, would you mind being more specific? I cannot see that issue. Thank you!

**Comment by marceloglacial** [comment]
Please, check out the video

https://user-images.githubusercontent.com/11657599/110566988-d061bb80-811e-11eb-8469-6e7c9e0d1d6d.mov

**Comment by dutraneto** [comment]
> Please, check out the video
> 
>  Screen.Recording.2021-03-09.at.9.30.33.PM.mov

Understood.

**Comment by dutraneto** [comment]
It is done

File: src/styles/components/location.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .location {
     background-color: var(--color-primary);
 
+    @include media('<=tablet') {
+        height: calc(100vh - 20.625rem);
```
Comment (marceloglacial) [inline src/styles/components/location.scss]: Solved for table. The issue persists on mobile. 

Screen Shot 2021-03-09 at 9 57 57 PM

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/styles/components/location.scss (hunk 1 above)
Comment (marceloglacial) [reply src/styles/components/location.scss]: TIP: add the header height to the section padding instead to change the height ;)

**Comment by dutraneto** [comment]
It should be working now. Thank you for your tip =)

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**